Dockerfile-operator

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM confluentinc/cp-kafka-connect:5.3.1
+ARG CP_VERSION
 
-ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"
+FROM confluentinc/cp-server-connect-operator:${CP_VERSION}.0
```

Comment:
I'll have to ask engineering / research regarding the additional version component and determine how and when it will rev.  In the interest of incremental improvements we could leave this for now and address when the situation arises.

Reply:
From engineering:

> As we do layering on the top of the CP images, the last number is for patches and bug fixes for the layered images.

So we might have to rev this version, but I didn't add a variable there figuring we could tackle that when a new version is released by them or do you want me to scaffold a variable for that as well?

Reply:
We can leave it for a future work item